src/Statistics.jl

```diff
@@ -167,12 +167,12 @@ _mean_promote(x::T, y::S) where {T,S} = convert(promote_type(T, S), y)
 
 function _mean(f, A::AbstractArray, dims=:)
     isempty(A) && return sum(f, A, dims=dims)/0
-    if dims == Colon()
+    if dims === (:)
```

Comment:
Parentheses are unnecessary.
```suggestion
    if dims === :
```

Reply:
Without brackets compiler gives the error
> ERROR: LoadError: syntax: space not allowed after ":" used for quoting

Reply:
Ah, sorry, I had checked `dims === 1` at the REPL and it works there. Not sure why not after `if`.